server/main.js

```diff
@@ -160,10 +160,18 @@ board.on("ready", function() {
     },
   });
 
-  board.analogRead("A0", function(raw) {
-    var mV = 5 * 1000 * (raw / 1024);
-    var value = (mV / 10) - 50;
-    temperatureCharacteristic.valueChange(value);
+  // Johnny-Five's Thermometer class provides a built-in
+  // controller definition for the TMP36 sensor. The controller
+  // handles computing a celsius (also fahrenheit & kelvin) from
+  // a raw analog input value.
+  // Reference: http://johnny-five.io/api/thermometer/
+  var temp = new five.Thermometer({
```

Comment:
Did you try this on the Edison, because I had code exactly like this and this.value was reporting undefined for me